### PR TITLE
format: detailed HEP record fixes

### PR DESCRIPTION
**Pull request by nikpap**
- [x] Rearranges the way external links are shown.

- [x] Restores cite record functionality.

- [x] Removes space before keywords.

- [x] Resolves error with `Uncaught Error: undefined missing js/citation_modal` when `ASSETS_DEBUG=False`

Signed-off-by: Nick Papoutsis <admin@nils.gr>

### CODE REVIEW

**Comment by jmartinm** [comment]
Does the Cite record function work? I get the following error in the console:

```javascript
TypeError: this.after is not a function(…)
```

**Comment by nikpap** [comment]
Nope, it's still a WIP. I've made it work now, but some further changes are required. I have a feeling this might need to be fixed in the brief results as well - the issue you describe is present there too.

File: inspire/base/templates/format/record/Inspire_Default_HTML_detailed.tpl [hunk 1]
```diff
@@ -73,4 +72,14 @@
       {{ record_citations() }}
     </div>
   </div>
+  <script type="text/javascript">
```
Comment (nikpap) [inline inspire/base/templates/format/record/Inspire_Default_HTML_detailed.tpl]: @jmartinm placing this in the `javascript` or `bundles` block doesn't seem to work. This template inherits from https://github.com/inveniosoftware/invenio-formatter/blob/master/invenio_formatter/templates/format/record/Default_HTML_detailed.tpl , which in turn inherits from https://github.com/inveniosoftware/invenio-formatter/blob/master/invenio_formatter/templates/format/record/Default_HTML_detailed_base.tpl. Neither of these have a `javascript` or `bundles` block.

**Comment by jmartinm** [comment]
I am getting an error in the console:

```Uncaught Error: Script error for "js/citation_modal"```

And `inveniomanage collect` does not seem to copy the file to the static/ folder. Can you remove the file from your folder `var/invenio.base-instance/static/js` and make sure that it is being correctly collected?

**Comment by nikpap** [comment]
Issue resolved, commit amended and rebased.

**Comment by jmartinm** [comment]
There is a problem with this PR when running in production mode. `ASSETS_DEBUG=False` in `invenio.cfg`

And error appears in the Javascript console.

**Comment by nikpap** [comment]
I'm trying to resolve the issue with `ASSETS_DEBUG=False`.

Does turning it to `True` resolve the issue in production mode?

**Comment by jmartinm** [comment]
Yes, `ASSETS_DEBUG=True` works. I can help you debug the problem.

**Comment by jmartinm** [comment]
The error will be solved when this is merged https://github.com/inspirehep/inspire-next/pull/607

Just need to add the citations file here https://github.com/inspirehep/inspire-next/pull/607/files#diff-3fc0e9ddbcfc37b1d8180a8bda46fc6dR24

**Comment by jmartinm** [comment]
Merged cf0633680f182229804678a391360fb5497a26a3

**Pull request closed without merging**